Adding link to API keys (#8209)
magento/devdocs: src/recommendations/implementation.md

   Deploying Product Recommendations requires two main data sources: [catalog and behavioral]({{ page.baseurl }}/recommendations/product-recs.html#types-of-data). Because production is the only environment where your shoppers' actions can captured and analyzed, it is in your best interest to start data collection on production as early as possible. You can expect higher quality recommendations when you have more data. As an added benefit, when you start collecting behavioral data on production, you can [fetch recommendations based on this production data while operating in non-production environments]({{ page.baseurl }}/recommendations/test.html). This allows you to test and experiment with different recommendations that are computed based on real shopper data collected in production.

>    To deploy data collection to production you need to [install and configure the Product Recommendations module]({{ page.baseurl }}/recommendations/install-configure.html) by [providing an API key](https://docs.magento.com/user-guide/system/saas.html#apikey).

Author: ```suggestion
   To deploy data collection to production, you need to [install and configure the Product Recommendations module]({{ page.baseurl }}/recommendations/install-configure.html) by [providing an API key](https://docs.magento.com/user-guide/system/saas.html#apikey).
```

>    To deploy data collection to production you need to [install and configure the Product Recommendations module]({{ page.baseurl }}/recommendations/install-configure.html) and [generate your API keys](https://docs.magento.com/user-guide/system/saas.html#apikey).

Reviewer: ```suggestion
   To deploy data collection to production you need to [install and configure the Product Recommendations module]({{ page.baseurl }}/recommendations/install-configure.html) by [providing an API key](https://docs.magento.com/user-guide/system/saas.html#apikey).
```

Reviewer: trying to avoid the double "and"
